scio-repl/src/main/scala/com/spotify/scio/repl/ReplScioContext.scala

```diff
@@ -24,6 +24,10 @@ class ReplScioContext(options: PipelineOptions,
                       artifacts: List[String])
   extends ScioContext(options, artifacts) {
 
+  private val scioReplName = "sciorepl"
+  this.setAppName(scioReplName)
+  this.setJobName(scioReplName)
```

Comment:
Actually I should include username there + some hash to get around limit of 1 jobname at a time.